src/java/us/kbase/mobu/initializer/ModuleInitializer.java

```diff
@@ -18,13 +20,25 @@
 
 public class ModuleInitializer {
 	public static final String DEFAULT_LANGUAGE = "python";
+	private static final String KB_DEP = "KB_SDK_CONFIG";
 
 	private String moduleName;
 	private String userName;
 	private String language;
-	private boolean verbose;
 	private File dir;
-	
+	private static final String CFG_LOC ="kb-sdk";
+	private boolean verbose;
+
+	// Setup a default configuration in case no config file is present
+	private static final Map<String,String> default_cfg;
+	static {
+		default_cfg = new HashMap<String,String>();
+		default_cfg.put("docker_image","kbase/sdkbase2"); // Base image name for use in module init Dockerfile
+		default_cfg.put("image_tag","v1.0"); // tag to use on the base image
+	}
+	// String array of required entries in the configuration object. Will throw exception if not found
+	static String[] required_configs = {"docker_image", "image_tag"};
```

Comment:
make final. Also consider doing

private final static List<String> requiredConfigs = Arrays.asList("x", "y");

Reply:
does this need to be accessible to other classes? If not make private.